src/AbstractPlotting.jl

```diff
@@ -239,15 +239,18 @@ export cgrad, available_gradients, showgradients
 
 export Pattern
 
+assetpath(files...) = normpath(joinpath(@__DIR__, "..", "assets", files...))
+
+export assetpath
```

Comment:
This clashes with GLMakie's assetpath (in a breaking way)

Reply:
Yeah, that's why I quickly wanted to get this in the current breaking release

Reply:
Well I mean breaking as in "it doesn't work". I pulled GLMakie and AbstractPlotting master maybe 30min ago now, and got a precompilation error and an error in AbstractPlottings `icon()` (it used GLMakies assetpath there).

Reply:
```julia
using GLMakie
fig = Figure()
[ Info: Precompiling GLMakie [e9467ef8-e4e7-5192-8a1a-b1aee30e663a]
WARNING: Method definition assetpath(Any...) in module AbstractPlotting at /home/frederic/.julia/dev/AbstractPlotting/src/AbstractPlotting.jl:242 overwritten in module GLMakie at /home/frederic/.julia/dev/GLMakie/src/GLMakie.jl:31.
  ** incremental compilation may be fatally broken for this module **

ERROR: IOError: readdir("/home/frederic/.julia/dev/GLMakie/src/GLVisualize/assets/icons"): no such file or directory (ENOENT)
...
  [4] icon()
    @ AbstractPlotting ~/.julia/dev/AbstractPlotting/src/AbstractPlotting.jl:248
```